community-stabilization-fund-api/src/modules/form-responses/components/FormResponsesTable.tsx

```diff
@@ -84,26 +84,89 @@ const FormResponsesTable: FC<FormResponsesTableProps> = ({
     FORM_RESPONSE_QUESTIONS
   ) as DataTableHeader<string>[];
 
-  useEffect(() => {
+  const filteredRows = (filterState: string[]): FormResponse[] => {
     if (filterState.length === 0) {
-      setFilteredFormResponses(formResponsesRef.current);
+      return formResponsesRef.current;
+    } else if (
```

Comment:
We could improve this by just filtering out whats in the filterState. Something like:

formRespRef.current.filter(f =>
  filterState.every(key => !!f[key] )
)